### PR TITLE
[SYCL] Add code owner for sycl-post-link

### PR DESCRIPTION
**Pull request by AlexeySachkov**
We don't have a particular code owner for this tool and at the moment it is covered by @bader as a generic code owner for the whole project.

If there are no objections, I would like to nominate myself for this role

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -86,3 +86,5 @@ sycl/doc/extensions/ @intel/dpcpp-specification-reviewers
 
 xpti/ @tovinkere @andykaylor
 xptifw/ @tovinkere  @andykaylor
+
+llvm/tools/sycl-post-link/ @AlexeySachkov
```
Comment (Fznamznon) [inline .github/CODEOWNERS:90]: I think we can also add @kbobrovs to list of sycl-post-link codeowners.
@kbobrovs , WDYT?

Comment (bader) [reply .github/CODEOWNERS:90]: +1 for adding @kbobrovs. It's recommended to have at least 2 code owners to back up during vacations, etc.

Comment (kbobrovs) [reply .github/CODEOWNERS:90]: sure, you can add me

Comment (AlexeySachkov) [reply .github/CODEOWNERS]: Done

**Pull request merged**